libs/community/langchain_community/embeddings/volcengine.py

```diff
@@ -24,19 +24,16 @@ class VolcanoEmbeddings(BaseModel, Embeddings):
     host: str = "maas-api.ml-platform-cn-beijing.volces.com"
     """host
     learn more from https://www.volcengine.com/docs/82379/1174746"""
+    
     region: str = "cn-beijing"
     """region
     learn more from https://www.volcengine.com/docs/82379/1174746"""
 
-    model: str = "bge-large-zh"
```

Comment:
self.model is referenced on line 116 I believe, I am not sure we can delete it as a param.

Reply:
lgtm other than this, @baskaryan